framework/src/auxkernels/VectorPostprocessorVisualizationAux.C

```diff
@@ -56,5 +65,8 @@ VectorPostprocessorVisualizationAux::timestepSetup()
 Real
 VectorPostprocessorVisualizationAux::computeValue()
 {
-  return _vpp_vector[_my_pid];
+  if (_use_broadcast)
+    return _vpp_vector[_my_pid];
```

Comment:
We should probably add a mooseAssert here. I don't readily see a way this can fail but it relies on parallel communication and proper order of execution. I'm ok not doing this if you really think it will _never_ break 😉.

Reply:
👍